photutils/aperture/circle.py

```diff
@@ -132,6 +132,22 @@ def __init__(self, positions, r):
         self.positions = self._sanitize_positions(positions)
         self.r = float(r)
 
+    def __repr__(self):
+        prefix = '<{0}('.format(self.__class__.__name__)
+        return '{0}{1}, r={2})>'.format(prefix, self._positions_str(prefix),
+                                        self.r)
+
+    def __str__(self):
+        prefix = 'positions'
+        clsinfo = [
+            ('Aperture', self.__class__.__name__),
```

Comment:
I wonder whether it would be useful to factor the common parts out to the baseclass and only append with the rest?